httpd-ldapauth-proxy/templates/deployment.yaml

```diff
@@ -45,22 +48,38 @@ spec:
             - name: config
               mountPath: /usr/local/apache2/conf/httpd.conf
               subPath: httpd.conf
+            - name: ldapconfigtemplate
+              mountPath: /run/config/httpdldaptemplate.conf
+              subPath: httpdldaptemplate.conf
+            - name: entrypoint
+              mountPath: /run/config/entrypoint.sh
+              subPath: entrypoint.sh
             - name: logs
               mountPath: /usr/local/apache2/logs/
-            - name: password
+          {{ if .Values.ldap.bindDN.enabled }}
```

Comment:
`ldap.bindDN.enabled` is a weird name for this value.

A DN is a path for an object in an LDAP tree. And "bind" is the name of the authentication process for ldap.

Reply:
Describing what it is used for not a wrong description of what you think it is :D 

If you translate bindDN into the HTTP Basic Auth language it would be "username for authentication" or something similar to that.

Reply:
Can you explicitly state what you want, that will be much easier/faster.